app/components/portfolios/details_component.rb

```diff
@@ -54,10 +54,39 @@ def all_subprojects
       all_descendants.project
     end
 
+    def render_sub_status_bar?
+      # When none of the descendants have a status set, there's nothing to show and we
+      # will return false.
+      sub_statuses.keys.any?(&:present?)
```

Comment:
As we expect either a string or `nil`, maybe:

```suggestion
      !sub_statuses.keys.all?(&:nil?)
```

Reply:
Hmm, do we think there could be a valid, sensible empty-string status? 🤔

I am assuming that when there is a status - it has a readable representation. So it has to be a non-blank string. `present?` will return false for `nil` and empty string and works nicely for this (or the opposite: `blank?`).

That leaves the question of: `!all?` or `any?`? I prefer `any` since it lets us avoid the not-operator - which I think is easy to miss by a glance.

Maybe I missed the point, please let me know if that's the case.